src/include/processor/operator/persistent/export_db.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "common/copier_config/csv_reader_config.h"
+#include "processor/operator/physical_operator.h"
+
+namespace kuzu {
+namespace processor {
+
+class ExportDB final : public PhysicalOperator {
+public:
+    ExportDB(std::string filePath, common::CSVOption copyToOption, uint32_t id,
+        const std::string& paramsString, std::vector<std::unique_ptr<PhysicalOperator>> children)
+        : PhysicalOperator{PhysicalOperatorType::EXPORT_DATABASE, std::move(children), id,
+              paramsString},
+          filePath(filePath), copyToOption{std::move(copyToOption)} {}
+
+    bool canParallel() const final { return false; }
+
+    bool isSource() const final { return true; }
```

Comment:
If you declare class as `final`, you don't need to declare function as `final`. I think it is recommended to declare function as `override` in such case.